code/game/machinery/telecomms/machines/relay.dm

```diff
@@ -23,7 +23,12 @@
 	// Add our level and send it back
 	var/turf/T = get_turf(src)
 	if(can_send(signal) && T)
-		signal.levels |= T.z
+		// Relays send signals to all ZTRAIT_STATION z-levels
+		if(SSmapping.level_trait(T.z, ZTRAIT_STATION))
+			for(var/z in SSmapping.levels_by_trait(ZTRAIT_STATION))
```

Comment:
better var names please :) https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names

Reply:
Fixed with da31852580b0e3c4a52e00579d8780baf8629ed1